core/src/main/java/org/mule/DefaultMuleContext.java

```diff
@@ -323,6 +329,73 @@ public synchronized void start() throws MuleException
         }
     }
 
+    private void startMessageSources() throws LifecycleException
+    {
+        startPipelineMessageSources();
+        startServicesMessageSources();
+    }
+
+    private void startServicesMessageSources() throws LifecycleException
+    {
+        for (Service service : this.getRegistry().lookupObjects(Service.class))
+        {
+            if (service.isStarted() || service.isPaused())
```

Comment:
What's the meaining of paused?. Wouldn't the source have to be paused and not started if the services is paused?